src/Components/Publishing/Layouts/ArticleWithFullScreen.tsx

```diff
@@ -90,3 +50,20 @@ export class ArticleWithFullScreen extends React.Component<
 }
 
 export default track()(ArticleWithFullScreen)
+
+export const getSlideshowImagesFromArticle = (article: ArticleData) => {
+  const fullscreenImages = []
+  let sectionIndex = 0
+
+  map(article.sections, section => {
+    if (includes(["image_collection", "image_set"], section.type)) {
+      map(section.images, image => {
```

Comment:
ditto

Reply:
good call, i just moved this logic from a different place... but it could use some improvements.